src/routes/navigation/TabNavigator/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import { MaterialIcons, Ionicons, FontAwesome5 } from "@expo/vector-icons";
+import { Colors } from "../../../configs/colors";
+
+import HomeView from "../../../views/HomeView";
+import NewPlantView from "../../../views/NewPlantView";
+import NotificationView from "../../../views/NotificationsView";
+import ProfileView from "../../../views/ProfileView";
+import LeafButton from "../../../components/LeafButton";
+
+const Tab = createBottomTabNavigator();
+
+const TabNavigator = (): React.ReactElement => {
+  return (
+    <Tab.Navigator>
+      <Tab.Screen
+        name="Home"
+        component={HomeView}
+        options={{
+          tabBarShowLabel: false,
```

Comment:
`tabBarShowLabel` can be set in `Tab.Navigator` so you don't need to re-set this in every `Tab.Screen`